### PR TITLE
Do not install docker and install podman

### PR DESCRIPTION
**Pull request by phracek**
Install distgen as soon as epel-release is also installed

Signed-off-by: Petr "Stone" Hracek <phracek@redhat.com>

### CODE REVIEW

File: roles/common_tools/tasks/main.yml [hunk 1]
```diff
@@ -39,16 +39,20 @@
     - python-pip
   when: releasever > 30
 
+- name: Install distgen package
```
Comment (zmiklank) [inline roles/common_tools/tasks/main.yml:42]: I think we do not really need distgen in TMT plans.
The distgen-generated files should be at this point correctly added to the repository.

Comment (phracek) [reply roles/common_tools/tasks/main.yml:42]: Fewer packages in TMT plans are always great. I will remote it.

Comment (phracek) [reply roles/common_tools/tasks/main.yml]: Resolved by https://github.com/sclorg/sclorg-testing-farm/pull/47/commits/1bbb41b40628b6885d2a7d38822539996d99b32d

File: plans/nightly-container-centos-7.fmf [hunk 2]
```diff
@@ -16,10 +16,5 @@ prepare:
       playbook:
         - tmt-testing-plan-centos7.yml
 
-    - name: Prepare CentOS for docker
```
[the remark this answers is not in the document]
Comment (phracek) [reply plans/nightly-container-centos-7.fmf:19]: This is already converted to ansible-playbook.
We can delete this and fix it centos7.fmf plan.
Or to have two plans till centos7.fmf plan will not be migrated.

**Pull request merged**